modules/virt-about-hostpath-provisioner.adoc

```diff
@@ -4,24 +4,23 @@
 
 :_content-type: CONCEPT
 [id="virt-about-hostpath-provisioner_{context}"]
-= About the hostpath provisioner
-
-The hostpath provisioner is a local storage provisioner designed for
-{VirtProductName}. If you want to configure local storage for
-virtual machines, you must enable the hostpath provisioner first.
-
-When you install the {VirtProductName} Operator, the hostpath provisioner Operator
-is automatically installed. To use it, you must:
-
-* Configure SELinux:
-** If you use {op-system-first} 8 workers, you must create a `MachineConfig`
-object on each node.
-** Otherwise, apply the SELinux label `container_file_t` to the persistent volume (PV) backing
-directory on each node.
-* Create a `HostPathProvisioner` custom resource.
-* Create a `StorageClass` object for the hostpath provisioner.
-
-The hostpath provisioner Operator deploys the provisioner as a _DaemonSet_ on each
-node when you create its custom resource. In the custom resource file, you specify
-the backing directory for the persistent volumes that the hostpath provisioner
-creates.
+= About the hostpath provisioner (HPP)
+
+ When you install the {VirtProductName} Operator, the Hostpath Provisioner Operator is automatically installed. The HPP is a local storage provisioner designed for {VirtProductName} that is created by the Hostpath Provisioner Operator. To use the HPP, you must create a HPP custom resource.
+
+[IMPORTANT]
+====
+In {VirtProductName} 4.10, the HPP Operator configures the Kubernetes CSI driver. The operator also recognizes the existing (legacy) format of the custom resource.
+
+The legacy HPP and the CSI host path driver are supported in parallel for a number of releases. However, at some point, the legacy HPP will no longer be supported. If you use the HPP, plan to create a storage class for the CSI driver as part of your migration strategy.
+====
+
+If you upgrade to {VirtProductName} version 4.10 on an existing cluster, the HPP operator is upgraded and the system does the following:
```

Comment:
"the following actions:"

(or similar)

Reply:
used _performs the following actions_